src/common/helpers.js

```diff
@@ -68,3 +68,24 @@ export function getAmazonDownloadLinkUrl(s3_bucket, s3_key) {
 export function getDomain() {
   return window.location.origin;
 }
+
+// Helper methods to ease working with ajax functions
+export const Ajax = {
```

Comment:
This is great! 👏 Can you refactor all the instances where we're doing requests?

Reply:
Sure! but do you think we should include that in this PR? It may be good to review this refactor separately to ensure we don't introduce any bugs.

Reply:
yeah, that works for me 👍